test/systemtests/docker_test.go

```diff
@@ -428,17 +428,21 @@ func (d *docker) cleanupContainers() error {
 	return d.node.tbnode.RunCommand("docker kill -s 9 `docker ps -aq`; docker rm -f `docker ps -aq`")
 }
 
-func (d *docker) startNetplugin(args string) error {
+func (d *docker) commonArgs() string {
 	netMode := d.node.suite.globInfo.Encap
 	fwdMode := d.node.suite.fwdMode
+	mode := "docker"
 	var storeArgs string
-	if cStore := d.node.suite.basicInfo.ClusterStore; strings.HasPrefix(cStore, "etcd") {
-		storeArgs = " --etcd-endpoints " + strings.Replace(cStore, "etcd", "http", 1) + " "
+	if d.node.suite.basicInfo.ClusterStoreDriver == "etcd" {
+		storeArgs = " --etcd-endpoints " + d.node.suite.basicInfo.ClusterStoreURLs + " "
 	} else {
-		storeArgs = " --consul-endpoints " + strings.Replace(cStore, "consul", "http", 1) + " "
+		storeArgs = " --consul-endpoints " + d.node.suite.basicInfo.ClusterStoreURLs + " "
```

Comment:
`d.node.suite.basicInfo.ClusterStoreURLs + " "` could be added to storeArgs outside the if block, not asking for change, just sayin